query-node/start.sh

```diff
@@ -20,11 +20,11 @@ docker-compose up -d db
 # Make sure we use dev config for db migrations (prevents "Cannot create database..." and some other errors)
 yarn workspace query-node config:dev
 
```

Comment:
Maybe it's due to my laptop being slow but sometime I get a `Error: Connection terminated unexpectedly` when running the `yarn workspace query-node-root start`. It's due to the db not being up yet. Relaunching the command once or at most twice works, but it might be confusing for someone new to the project.

Would this be acceptable fix?
```suggestion

# Wait for the db to be up
until docker-compose logs db | grep "ready for start up"; do
  echo "Waiting for the db to be ready..."
  sleep 1
done

```

Reply:
Looks good, thank you